### DISCUSSION TITLE
Security Mastering Enablement

### DISCUSSION THREAD

**Discussion by jtylwalk** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#110] [Suggestions] [answered]
One area that came up while reviewing the spec is around security mastering: currently the `security_id` field is defined as being a free text string, but it would be great if there was a viewpoint around a future state that could provide or enable security mastering consistently across the industry; I would recognize that this is maybe a bit outside the intentions of the spec as it currently stands.

There are standards bodies and well-known identifiers that are leveraged throughout other areas of finance (CUSIPs, RICs, SEDOLs, etc), and many of these have consistent formatting for their associated identifiers. Could this spec be used to also push towards making progress in the dimension of a consistent identification, in this space?

There is also that idea of possibly deepening the ability to tag multiple identifiers to a security in the spec, so that one could map different identifiers into a single security identifier, which would be used throughout the rest of the feed files.

**Comment by JSv4** [comment, marked as answer]
Thanks for the suggestion @jtylwalk. Given our current release timeline and bandwidth, this is definitely out of scope for our first public release. It's an interesting idea. In addition to possibly requiring some changes to the spec, as you point out, it seems it would also require some agreement from various market participants, which would probably be much harder. This could be an interesting discussion for the proposed standards group, which was conceived as a way to try to drive industry standardization and practices where a purely technical solution isn't a sufficient solution. 

Maybe the most expedient way to solve this from a technical perspective is to have a "SecurityIdentifier" type and allow any given security to have a list of such identifiers, again, as you point out. For private company securities, are you aware of any agreed identifiers or standards that would benefit from such a change?

**Reply by jtylwalk** [reply]
Hey - thanks for the response; I was actually in an OCF tooling subcommittee meeting on Wednesday (Kathryn Montalvo's meeting) - and we discussed a bit around this issue there as well.

I mentioned that we do elements of matching / mastering for private companies using identifiers from the thirdparties we interact with - primarily PitchBook, Preqin, and Zanbato. There's also other well-known ids that could be leveraged: DUNS number, tax identifiers, etc.
